source/_posts/2022-12-07-release-202212.markdown

```diff
@@ -443,6 +444,41 @@ The following integrations are now available via the Home Assistant UI:
 [abode docs]: /integrations/abode/
 [accuweather docs]: /integrations/accuweather/
 
+## Release 2022.12.4 - December 12
+
+- Bump bluetooth-auto-recovery to 1.0.0 ([@bdraco] - [#83800]) ([bluetooth docs])
+- Update frontend to 20221212.0 ([@bramkragten] - [#83842]) ([frontend docs])
+- Bump version of ZHA quirks to 0.0.89 ([@dmulcahey] - [#83848]) ([zha docs])
+- Bump pychromecast to 13.0.3 ([@emontnemery] - [#83861]) ([cast docs])
+- Move template check into fritzbox entry setup ([@mib1185] - [#83863]) ([fritzbox docs])
+- Use the async_migrate_paypal_agreement function to get the migration URL ([@ludeeus] - [#83469]) ([cloud docs])
+
+[#83469]: https://github.com/home-assistant/core/pull/83469
+[#83482]: https://github.com/home-assistant/core/pull/83482
+[#83592]: https://github.com/home-assistant/core/pull/83592
+[#83778]: https://github.com/home-assistant/core/pull/83778
+[#83797]: https://github.com/home-assistant/core/pull/83797
+[#83800]: https://github.com/home-assistant/core/pull/83800
+[#83842]: https://github.com/home-assistant/core/pull/83842
+[#83848]: https://github.com/home-assistant/core/pull/83848
+[#83861]: https://github.com/home-assistant/core/pull/83861
+[#83863]: https://github.com/home-assistant/core/pull/83863
+[@balloob]: https://github.com/balloob
+[@bdraco]: https://github.com/bdraco
+[@bramkragten]: https://github.com/bramkragten
+[@dmulcahey]: https://github.com/dmulcahey
+[@emontnemery]: https://github.com/emontnemery
+[@frenck]: https://github.com/frenck
+[@mib1185]: https://github.com/mib1185
+[abode docs]: /integrations/abode/
+[accuweather docs]: /integrations/accuweather/
+[bluetooth docs]: /integrations/bluetooth/
+[cast docs]: /integrations/cast/
+[cloud docs]: /integrations/cloud/
+[fritzbox docs]: /integrations/fritzbox/
+[frontend docs]: /integrations/frontend/
+[zha docs]: /integrations/zha/
```

Comment:
## Release 2022.12.4 - December 12

- Bump bluetooth-auto-recovery to 1.0.0 ([@bdraco] - [#83800]) ([bluetooth docs])
- Update frontend to 20221212.0 ([@bramkragten] - [#83842]) ([frontend docs])
- Bump version of ZHA quirks to 0.0.89 ([@dmulcahey] - [#83848]) ([zha docs])
- Bump pychromecast to 13.0.3 ([@emontnemery] - [#83861]) ([cast docs])
- Move template check into fritzbox entry setup ([@mib1185] - [#83863]) ([fritzbox docs])
- Use the async_migrate_paypal_agreement function to get the migration URL ([@ludeeus] - [#83469]) ([cloud docs])
- Fix issue on Overkiz Domestic Hot water heater entities with away mode ([@nyroDev] - [#83684]) ([overkiz docs])
- Bump pyoverkiz to 1.7.2 ([@nyroDev] - [#83866]) ([overkiz docs])
- Fix bleak-retry-connector get_device hang with HAOS 9.4 and only proxies ([@bdraco] - [#83879]) ([bluetooth docs])
- Fix HomeKit media players when entity has duplicate sources ([@bdraco] - [#83890]) ([homekit docs])

[#83469]: https://github.com/home-assistant/core/pull/83469
[#83482]: https://github.com/home-assistant/core/pull/83482
[#83592]: https://github.com/home-assistant/core/pull/83592
[#83684]: https://github.com/home-assistant/core/pull/83684
[#83778]: https://github.com/home-assistant/core/pull/83778
[#83797]: https://github.com/home-assistant/core/pull/83797
[#83800]: https://github.com/home-assistant/core/pull/83800
[#83842]: https://github.com/home-assistant/core/pull/83842
[#83848]: https://github.com/home-assistant/core/pull/83848
[#83861]: https://github.com/home-assistant/core/pull/83861
[#83863]: https://github.com/home-assistant/core/pull/83863
[#83866]: https://github.com/home-assistant/core/pull/83866
[#83879]: https://github.com/home-assistant/core/pull/83879
[#83890]: https://github.com/home-assistant/core/pull/83890
[@balloob]: https://github.com/balloob
[@bdraco]: https://github.com/bdraco
[@bramkragten]: https://github.com/bramkragten
[@dmulcahey]: https://github.com/dmulcahey
[@emontnemery]: https://github.com/emontnemery
[@frenck]: https://github.com/frenck
[@ludeeus]: https://github.com/ludeeus
[@mib1185]: https://github.com/mib1185
[@nyroDev]: https://github.com/nyroDev
[abode docs]: /integrations/abode/
[accuweather docs]: /integrations/accuweather/
[bluetooth docs]: /integrations/bluetooth/
[cast docs]: /integrations/cast/
[cloud docs]: /integrations/cloud/
[fritzbox docs]: /integrations/fritzbox/
[frontend docs]: /integrations/frontend/
[homekit docs]: /integrations/homekit/
[overkiz docs]: /integrations/overkiz/
[zha docs]: /integrations/zha/